models/release.go

```diff
@@ -134,6 +135,8 @@ func createTag(gitRepo *git.Repository, rel *Release) error {
 		if err != nil {
 			return fmt.Errorf("CommitsCount: %v", err)
 		}
+	} else {
+		rel.CreatedUnix = util.TimeStamp(time.Now().Unix())
```

Comment:
change to `util.TimeStampNow()`?

Reply:
Thanks, updated and the CI passed as well.